web/src/people.ts

```diff
@@ -763,7 +764,7 @@ export function emails_to_slug(emails_string: string): string | undefined {
     if (emails.length === 1 && emails[0] !== undefined) {
         const person = get_by_email(emails[0]);
         assert(person !== undefined, "Unknown person in emails_to_slug");
-        slug += get_slug_from_full_name(person.full_name);
```

Comment:
I feel like the bug might be that `get_slug_from_full_name` should handle encoding?